### ISSUE TITLE
[Qualify of Life] config editor opens on first boot

### ISSUE THREAD

**Issue by techyCoder81** [Raytwo/ARCropolis#263]
### Have you read the disclaimer before proceeding?

- [X] I have read the disclaimer.

### Explain what could get improvements.

Opening the config editor on first boot would help some troubleshooting instances. Specifically, region crashes are a problem for some users that is hard to track down at times. Making region selection more accessible in any way would be huge. 

### How could we make this better?

If the config editor opened on first boot, people might have a better chance at changing their region easier before having cashes.

**Comment by techyCoder81** [comment]
so heres the flowchart
```
japanese user installs hdr successfully
   japanese user did NOT change region in arcrop
      japanese user has to take out hdr-assets from the sd
      inexperienced user will manually turn off switch instead of mount in hekate
      take sd out, plug it in, move hdr-assets one directory up
      reinstert, re RCM and inject, launch smash, change region
      wait for a reboot, confirm the setting is changed
      turn switch off, take sd out, move hdr-assets back down
      reinstert, re RCM and inject, launch smash
            enjoy hdr

japanese user installs hdr successfully
   japanese user miraculously has JP set for region
       press play on launcher
            enjoy hdr

keep in mind this is also way worse for EUR users living in somewhere like germany, who have smash in english, but try EUR German first, and have to repeat every thing in the first paragraph again because they dont know better
```

**Comment by Raytwo** [comment]
Implemented as of ARCropolis 3.2.0
